app/src/components/Forms/Info/DetailedInfo.js

```diff
@@ -0,0 +1,247 @@
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                program website
+              </a>{' '}
+              for full details.
+            </i>
+          </p>
+          <ul>
+            <li>Candidates may opt into the program.</li>
+            <li>
+              To qualify, they must show that they have{' '}
+              <u>broad community support</u> by collecting 250 small
+              contributions from Portlanders if running for Council or Auditor
+              or 500 small contributions if running for Mayor.
+            </li>
+            <li>
+              Participating candidates must agree to contribution limits. They
+              must{' '}
+              <u>
+                agree to collect no more than $250 per donor and only from
+                humans, not businesses or political committees
+              </u>
+              , with two exceptions:
+              <ul>
+                <li>
+                  The first exception is that candidates may raise as much as
+                  $5,000 in “seed money” in order to raise their 250 or 500
+                  qualifying contributions. This helps them hire a treasurer and
+                  campaign manager or print materials and build a website to
+                  collect small contributions. This money can come from a past
+                  campaign, themselves as a gift or a loan, or from donors.
+                </li>
+                <li>
+                  The second exception is $20,000 of in-kind contributions. This
+                  enables candidates to get <i>non-monetary</i> support – like
+                  throwing house parties or creating art for their campaign –
+                  from supporters, up to a total campaign limit of $20,000.
+                </li>
+              </ul>
+            </li>
+            <li>
+              As candidates collect small contributions from Portlanders, the
+              City matches them 6-to-1 on the first $50. So if you give a
+              candidate $10 a month, the City will match that with $60 each time
+              until you’ve given $50 and the City has provided the candidate
+              $300 in matching funds for you.
+            </li>
+            <li>
+              You can get matched for each race, so you could give to someone
+              running for Mayor and get matched, and to candidates in each of
+              the Commission races and get matched, and to a candidate in the
+              Auditor’s race and get matched.
+            </li>
+          </ul>
+          <h3>What it's for</h3>
+          <p>
+            Our government works best when every person is engaged in the
+            elections process and can make a difference in the community. The
+            program fosters an inclusive democratic process where everyone
+            participates and everyone’s contributions matter. A community where
```

Comment:
"A community where..." sentence isn't a sentence, just 2 objective clauses and a comma

Reply:
Added "we support a"